tests/TextDiffTest.php

```diff
@@ -93,7 +93,7 @@ public function testAllFormatsCovered()
         sort($formats);
 
         $this->assertSame(
-            ['json', 'junit', 'txt', 'xml'],
+            ['checkstyle', 'json', 'junit', 'txt', 'xml'],
```

Comment:
please add `checkstyle` to this provider `https://github.com/FriendsOfPHP/PHP-CS-Fixer/pull/3119/files#diff-51abbc422b4e427c2f931b47ead68b52R74` as well

Reply:
As the *checkstyle* format does not include diffs, I don't think adding it to the test you mentioned is relevant :)

Reply:
ah yes, basically the test I think would be nice to have is to check that the [OutputFormatter::escape($dom->saveXML())](https://github.com/FriendsOfPHP/PHP-CS-Fixer/pull/3119/files#diff-cea3208f83df804b871237e6ed98b215R57) is used (your code is correct, a test to prevent regression would be nice).

Reply:
I'll add a test today, then :)